client/dist/index.html

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
+
 <title>Chart Module</title>
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 <link rel="stylesheet" href="style.css">
-<div id="app">Hello World!</div>
```

Comment:
Still a little confused about your index.html and styles.css files, as I see you referencing the body in styles.css, but there's no body element in your html file.

Reply:
@seungprk

Reply:
I will add the html/head/body tags in. I was following the google html style guide which says to omit html/head/body tags but upon further research it seems that while modern browsers don't have issues with this, most people expect the aforementioned tags and there may be issues with libraries expecting them. Also I agree it is confusing that I reference body in css but I don't explicitly state it in the html.